README.md

```diff
@@ -68,18 +67,29 @@ Note: This is an example of a non-normative note.
 
 # Overview
 
-Serving GraphQL over HTTP provides the ability to use the full advantages of GraphQL with the rich feature set of HTTP. Carrying GraphQL in HTTP does not mean that GraphQL overrides existing
-semantics of HTTP but rather that the semantics of GraphQL over HTTP map naturally to HTTP semantics.
+Serving GraphQL over HTTP provides the ability to use the full advantages of GraphQL with the rich feature set of HTTP.
+Carrying GraphQL in HTTP does not mean that GraphQL overrides existing semantics of HTTP,
+but rather that the semantics of GraphQL over HTTP map naturally to HTTP semantics.
 
-GraphQL naturally follows the HTTP request/response message model providing a GraphQL request in an HTTP request and
-GraphQL response in an HTTP response.
+GraphQL naturally follows the HTTP request/response message model,
+providing a GraphQL request in an HTTP request and GraphQL response in an HTTP response.
 
 # URL
 
-A GraphQL server operates on a single URL and all GraphQL requests for a given service should be directed
-at this URL. Other protocols may also use that URL.
+A GraphQL over HTTP compliant server MUST designate at least one URL that handles GraphQL requests.
+
+It is allowed for the same GraphQL schema to be available on multiple URLs.
+
+All GraphQL operations that are available in the schema MUST be available on each designated URL.
+That means the client can send all their GraphQL requests to a single endpoint, regardless
+of the contained Operations, Fragments or Variables.
+
+Other protocols may also use those URLs.
```

Comment:
I'm not sure `protocols` is the right word here.

Reply:
I expanded that bit.